hbase-common/src/main/java/org/apache/hadoop/hbase/TableName.java

```diff
@@ -192,16 +192,16 @@ public static void isLegalTableQualifierName(final byte[] qualifierName,
     if(end - start < 1) {
       throw new IllegalArgumentException(isSnapshot ? "Snapshot" : "Table" + " qualifier must not be empty");
     }
+    // Treat the bytes as UTF-8
+    String qualifierString = new String(
```

Comment:
nit: Why do this 'new String' instead of doing Bytes.toString(qualifierName, start, end - start) ?

I fixed the summary.

Reply:
Thanks!!!
No special reason here. Just copy the origin code.
Now I changed code to use Bytes.toString(qualifierName, start, end - start) instead of new String. 
Is this better?